Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -75,18 +74,24 @@ def parser(cls, parser, loggers=None):
     @classmethod
     def get_commit_info(cls, args, repository, **kwargs):
         commit_objects = []
-        commits_to_revert = []
         commit_bugs = set()
+        commit_radars = set()
```

Comment:
What does this have to do with `--unsafe`?

Reply:
Oops sorry I had a bunch of merge conflicts. Just resolved them!